lib/find.js

```diff
@@ -102,7 +102,13 @@ function _find(tree, _options) {
   var options, root;
 
   if (arguments.length === 1) {
-    var decomposition = decompose(tree);
+    var decomposition;
+
+    if (typeof tree === 'object') {
+      decomposition = decompose(tree.inputTree);
```

Comment:
I'm not sure I fully understand why this was required, so I wasn't sure what to suggest as an alternative.